modules/plottinggl/include/modules/plottinggl/processors/parallelcoordinates/parallelcoordinates.h

```diff
@@ -137,6 +140,8 @@ class IVW_MODULE_PLOTTINGGL_API ParallelCoordinates : public Processor {
     FloatProperty falllofPower_;
     FloatProperty lineWidth_;
     FloatProperty selectedLineWidth_;
+    FloatProperty axisWidth_;
+    FloatProperty selectedAxisWidth_;
```

Comment:
Would it be possible to just have a thicker axis for picking , i.e. not visually? Then we could skip these properties and use a heuristic/hard coded values instead. 
Less is more :)

Reply:
We could. I just haven't figured it out in a nice way just yet ;)

Do we want the picking axis to be wider by a ratio or by a fix set of pixels?

Reply:
Can you check if the fragment Coord is more than one pixel from the center of the line and draw transparent color in that case?

Probably want it to be at least 3 pixels wide? If the axis is wider there is no need to extend the picking.